.github/workflows/run-tests.yml

```diff
@@ -86,16 +86,12 @@ jobs:
         python -m pip install --break-system-packages --upgrade pip
         pip install -r requirements.txt
         pip install pytest
-    - name: test (macos)
-      if: ${{ matrix.os == 'macos-latest' }}
-      env:
-        PYTHONWARNINGS: default
-      run: |
-        . "$HOME/.local/share/venvs/inkscape/bin/activate"
-        pytest -s -vv test
-    - name: test (non-macos)
-      if: ${{ matrix.os != 'macos-latest' }}
+    - name: test
       env:
         PYTHONWARNINGS: default
       run: |
-        pytest -s -vv test
+       set -e
+       if [[ "$RUNNER_OS" == "macOS" ]]; then
```

Comment:
@sam-itt could you detect if this file exists - and if so source it? ...so it will work platform independently